server/dals/network-dal.js

```diff
@@ -51,8 +51,8 @@ const buildQueryByType = function (queryType, query) {
         return buildNetworkSourceQuery();
     case "NetworkGeneFromSource":
         return buildNetworkGeneFromSourceQuery(query.gene, query.source, query.timestamp);
-    case "CreateNetwork":
-        return buildCreateNetworkQuery(query.genes, query.source, query.timestamp);
+    case "GenerateNetwork":
+        return buildGenerateNetworkQuery(query.genes, query.source, query.timestamp);
     }
```

Comment:
This is a `switch` with no `default`: is it intentional for this function to return `undefined` if none of the cases match?

Reply:
Data-driven design is probably called for here too

Reply:
I fixed that as well